packages/firestore/src/api/database.ts

```diff
@@ -447,6 +447,39 @@ export class Firestore implements firestore.FirebaseFirestore, FirebaseService {
     return deferred.promise;
   }
 
+  /**
+     * Shuts down this Firestore instance.
```

Comment:
leading spaces seems off...  If you run `yarn lint` from packages/firestore, it should complain about this (and it's complaining in the CI build too).